<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GlobalAudioConfiguration</class>
 <widget class="QWidget" name="GlobalAudioConfiguration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>367</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>361</width>
     <height>341</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="2">
       <widget class="QComboBox" name="m_pCbCurrentSyncFactor"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="m_pLblCurrentAudioFramesBuffered">
        <property name="text">
         <string>Current Number of Audio Frames Buffered</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="m_pLblNumberAudioPorts">
        <property name="text">
         <string>Number of Audio Ports</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="m_pEditNumberAudioPorts">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="inputMask">
         <string notr="true"/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QComboBox" name="m_pCbCurrentAudioFramesBuffered"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="m_pLblCurrentSyncFactor">
        <property name="text">
         <string>Current Sync Factor</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="m_pTblAudioConfigurations">
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Configuration Number</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Bit Depth</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Sample Rate</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
